examples/s3_static_files/README.md

```diff
@@ -10,7 +10,7 @@ TLDR:
 
 ```shell
 echo "nautobot[remote_storage]" >> $NAUTOBOT_ROOT/local_requirements.txt
-pip3 install "nautobot[remote_storage]"
+pip3 install "nautobot[remote_storage]" boto3
```

Comment:
I noticed that installing `nautobot[remote_storage]` doesn't actually include `boto3` by default, I guess because `django-storages` supports many different backends, not just S3.